bot/code_review_bot/__init__.py

```diff
@@ -100,6 +107,45 @@ def is_publishable(self):
 
         raise Exception("Unsupported publication mode {}".format(settings.publication))
 
+    def build_hash(self):
+        """
+        Build a uniquely identifying hash for that issue
+        """
+        assert self.revision is not None, "Missing revision"
+
+        try:
+            # Load all the lines affected by the issue
+            file_content = self.revision.load_file(self.path)
+        except requests.exceptions.HTTPError as e:
+            if e.response.status_code == 404:
+                logger.warning(
+                    "Failed to download a file with an issue", path=self.path
+                )
+                return
+            raise
+
+        # Build raw content:
+        # 1. lines affected by patch
+        # 2. without any spaces around each line
+        file_lines = file_content.splitlines()
+        start = self.line - 1  # file_lines start at 0, not 1
+        raw_content = "\n".join(
+            [
+                line.strip()
+                for line in file_lines[start : start + self.nb_lines]  # noqa E203
```

Comment:
Nit: Let's blindly follow the coding style enforced by black, that's why we have black after all ;)

Reply:
This is the coding style from black !
The noqa is to pease flake8 too...

Reply:
We need to add `ignore = E203` to the .flake8 file (https://github.com/psf/black#slices).